### DISCUSSION TITLE
Select separate channels for parent / child deliveries (sub packs to different channel)

### DISCUSSION THREAD

**Discussion by Decollates** [gameserverapp/Platform#374] [Suggestions]
OK SO! 

This suggestion deals with the discord purchase feed that is made when deliveries are completed. At this time, the discord relays all feed regarding deliveries to a single channel. 

By having the option to select specific channels for the delivery messages of sub packs and packs, some security can be maintained over the nature of some deliveries (sales come to mind).

**Comment by ad3d0** [comment]
You can always just add "discord message" action to pack and subpacks themself. This way you can set different channel and even custom message for each pack separately. Here is an example from GSA and how it looks in Discord.

![Screenshot_969](https://user-images.githubusercontent.com/43604113/225933100-f9f956d9-b666-4fb0-9376-073a1fd6257f.png)
![Screenshot_970](https://user-images.githubusercontent.com/43604113/225933106-bb3c14c4-8df0-4c04-96cb-33d246496532.png)

**Comment by mvaessen** [comment]
Closing because provided alternative should suffice
